services/order.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional
+from datetime import datetime
+from django.db import transaction
+from django.db.models.query import QuerySet
+from db.models import Ticket, Order, User, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: Optional[datetime] = None
+) -> None:
+    user = User.objects.get(username=username)

```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency). Fix it everywhere)